app/components/generic/tabs/BasicTabs.tsx

```diff
@@ -38,20 +39,28 @@ export default function BasicTabs({ tabs, children }: { tabs: ITab[]; children?:
         <div className="border-b border-gray-200">
           <div className="w-full -mb-px flex justify-between items-center" aria-label="Tabs">
             <div className=" -mb-px flex justify-start">
-              {tabs.map((tab) => (
-                <Link
-                  href={tab.href}
-                  key={tab.name}
-                  className={classNames(
-                    'first:ml-0 lg:ml-20 w-50 py-5 text-center text-lg font-bold',
-                    compareUrlsIgnoreLastSegments(tab.href, pathname, tab.ignoreSegments ?? 0)
-                      ? "relative border-bcorange text-bcblue before:content-[''] before:absolute before:w-full before:border-b-3 before:border-bcorange before:bottom-0 before:left-1/2 before:-translate-x-1/2"
-                      : "relative border-transparent text-gray-300 hover:before:content-[''] hover:before:absolute hover:before:w-full hover:before:border-b-3 hover:before:border-gray-300 hover:before:bottom-0 hover:before:left-1/2 hover:before:-translate-x-1/2",
-                  )}
-                >
-                  {tab.label}
-                </Link>
-              ))}
+              {tabs.map((tab) => {
+                const linkingClassNames = classNames(
+                  'first:ml-0 lg:ml-20 w-50 py-5 text-center text-lg font-bold',
+                  compareUrlsIgnoreLastSegments(tab.href, pathname, tab.ignoreSegments ?? 0)
+                    ? "relative border-bcorange text-bcblue before:content-[''] before:absolute before:w-full before:border-b-3 before:border-bcorange before:bottom-0 before:left-1/2 before:-translate-x-1/2"
+                    : "relative border-transparent text-gray-300 hover:before:content-[''] hover:before:absolute hover:before:w-full hover:before:border-b-3 hover:before:border-gray-300 hover:before:bottom-0 hover:before:left-1/2 hover:before:-translate-x-1/2",
+                );
+
+                const linkingComponent = (
+                  <Link href={tab.href} key={tab.name} className={linkingClassNames}>
+                    {tab.label}
+                  </Link>
+                );
+
+                return tab.name === 'comments' ? (
```

Comment:
Let's generalize the tab functionality by adding a `tooltip: string` prop to `ITab`. If `tab.tooltip` is not null, the tooltip will display the provided string value.